data.sql

```diff
@@ -16,3 +16,6 @@ UPDATE animals SET owner_id = 2  WHERE name = 'Gabumon' OR name ='Pikachu';
 UPDATE animals SET owner_id = 3  WHERE name = 'Devimon' OR name ='Plantmon';
 UPDATE animals SET owner_id = 4  WHERE name = 'Charmander' OR name ='Squirtle' OR name = 'Blossom';
 UPDATE animals SET owner_id = 5  WHERE name = 'Angemon' OR name ='Boarmon';
+
+INSERT INTO vets (name, age, date_of_graduation ) VALUES ('Vet William Tatcher', 45, '2000/4/23' ), ('Vet Maisy Smith', 26, '2019/01/17'), ('Vet Stephene Mendez', 64, '1981/05/04'), ('Vet Jack Harkness', 38, '2008/06/08');
+ INSERT INTO visits VALUES (1, 1, '2020/05/24'), (1, 3, '2020/06/22'), (2, 4, '2021/02/02'), (3, 2, '2020/01/05'), (3,2, '2020/03/08'), (3,2, '2020/05/14'), (4, 3, '2021/05/04'), (5, 4, '2021/02/24'), (6, 1, '2019/12/21'), (6, 2, '2021/04/07'), (6, 2, '2020/08/10'), (7, 3, '2019/09/29'), (8, 4, '2020/10/03'), (8, 4, '2020/11/04'), (12, 2, '2019/01/24'), (12, 2, '2019/05/15'), (12,2, '2020/02/27'), (12,2, '2020/08/03'), (10, 2, '2020/05/24'), (10, 1, '2021/01/11');
```

Comment:
- [ ] Please make sure that your data insertion information has
![Screenshot from 2022-10-20 23-13-56](https://user-images.githubusercontent.com/6615394/197076923-9abc1610-6164-40a8-b1b0-32043386649f.png)
![Screenshot from 2022-10-20 23-13-44](https://user-images.githubusercontent.com/6615394/197076933-1ebfaa64-1686-472c-9972-6f5813820ff7.png)
 the same properties as the table properties in order to avoid errors like the one below :arrow_down: 
![Screenshot from 2022-10-20 23-14-02](https://user-images.githubusercontent.com/6615394/197076898-38fd51cd-4a86-4328-8a39-e918583d3a53.png)

Reply:
]you have to execute the query `ALTER TABLE animals ADD CONSTRAINT fk_species_id FOREIGN KEY (species_id) REFERENCES species (id);` which is used to add species_id to the animals table .
It was one of the exercise for Tuesday to make this changes on the animal table